### PR TITLE
Connections to BE

### PR DESCRIPTION
**Pull request by tomaskk**
login/register pages
management --> users tab fully functional

### CODE REVIEW

File: src/components/LoginPage/LoginPage.jsx [hunk 1]
```diff
@@ -2,13 +2,15 @@
 import SvgLogo from "./LoginPageComponents/img/SvgLogo.jsx";
 import SlackLogo from "./LoginPageComponents/img/SlackLogo.jsx";
 import Illustration from "./LoginPageComponents/img/illustration.png";
-const ServerHostName = 'http://localhost:8080';
+const ServerHostName = 'http://localhost:5001';
```
Comment (WarHatch) [inline src/components/LoginPage/LoginPage.jsx:5]: Didn't you set this in a file somewhere? :D

**Review by WarHatch — APPROVED** [review]
One smol thing to fix later in your work, but still a good PR

File: src/components/LoginPage/LoginPage.jsx (hunk 1 above)
Comment (WarHatch) [reply src/components/LoginPage/LoginPage.jsx:5]: @tomaskk  a ping for good measure

Comment (tomaskk) [reply src/components/LoginPage/LoginPage.jsx:5]: @WarHatch yeah we got it in `src/constants/serverUri.js`

That's just me being lazy

**Pull request merged**